bin/install-dependencies.sh

```diff
@@ -16,7 +16,8 @@ pycsw_tag='2.4.0'
 
 pipopt='--exists-action=b'
 
-$pip install $pipopt -U pip
+curl https://bootstrap.pypa.io/2.7/get-pip.py -o get-pip.py
+source venv/bin/activate && python get-pip.py pip==20.3.4
```

Comment:
Which `venv` is this because it doesn't appear to be created in this script?

Reply:
it's created in the `Jenkinsfile` - https://github.com/alphagov/ckanext-datagovuk/blob/master/Jenkinsfile#L18

Reply:
Hmm I'm wondering if the `pip` installation belongs in the `Jenkinsfile` before calling `install-dependencies.sh` because the `pip` binary is passed in as the first argument (https://github.com/alphagov/ckanext-datagovuk/blob/5347a33244e09ab25e0ff097b8f5f7e9af454baa/bin/install-dependencies.sh#L3) so it feels like that should have happened prior to this script being called.

Especially given that this script is currently generic enough that it doesn't matter what sort of virtual enviroenment you've got set up, you just need to specify which `pip` to use, but now this becomes coupled to the environment set up by the `Jenkinsfile`.

Reply:
sure, could move it there